packages/elements/src/item/index.ts

```diff
@@ -272,7 +326,7 @@ export class Item extends ControlElement {
         ${this.multipleTemplate}
         <slot name="left"></slot>
       </div>
-      <div part="center" id="label">
+      <div part="center" id="label" ${ref(this.labelRef)}>
```

Comment:
Remove the element id now that you are using ref

Reply:
Updated.